mayastor/src/bdev/nvmx/controller.rs

```diff
@@ -393,11 +393,11 @@ impl<'a> NvmeController<'a> {
         // Reset the controller to complete all remaining I/O requests after all
         // I/O channels are closed.
         // TODO: fail the controller via spdk_nvme_ctrlr_fail() upon shutdown ?
-        debug!("{} resetting NVMe controller", ctx.name);
-        let rc = unsafe { spdk_nvme_ctrlr_reset(controller.ctrlr_as_ptr()) };
-        if rc != 0 {
-            error!("{} failed to reset controller, rc = {}", ctx.name, rc);
-        }
+        //debug!("{} resetting NVMe controller", ctx.name);
+        //let rc = unsafe { spdk_nvme_ctrlr_reset(controller.ctrlr_as_ptr()) };
+        //if rc != 0 {
+        //    error!("{} failed to reset controller, rc = {}", ctx.name, rc);
+        //}
```

Comment:
Why is it no longer necessary to reset controllers on shutdown?

Reply:
I doubt it ever was.